tests/Unit/Model/Widget/SpanParserTest.php

```diff
@@ -0,0 +1,278 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PhpTui\Tui\Tests\Unit\Model\Widget;
+
+use InvalidArgumentException;
+use PhpTui\Tui\Model\Color\AnsiColor;
+use PhpTui\Tui\Model\Color\RgbColor;
+use PhpTui\Tui\Model\Modifier;
+use PhpTui\Tui\Model\Widget\SpanParser;
+use PHPUnit\Framework\TestCase;
+
+class SpanParserTest extends TestCase
+{
+    public function testParseOneTag(): void
+    {
+        $spans = SpanParser::new()->parse('<fg=green bg=blue options=bold,italic>Hello</> World');
```

Comment:
as mentioned previously, to be compatible with Symfony we need this to be delimited with `;`

Reply:
✅